postgres/postgres.go

```diff
@@ -56,6 +56,12 @@ func New(config ...Config) *Storage {
 	if cfg.Username != "" || cfg.Password != "" {
 		dsn += "@"
 	}
+	// unix socket host path
+	if strings.HasPrefix(cfg.Host, "/") {
+		dsn += fmt.Sprintf("%s:%d", cfg.Host, cfg.Port)
+	} else {
+		dsn += fmt.Sprintf("%s:%d", url.QueryEscape(cfg.Host), cfg.Port)
+	}
 	dsn += fmt.Sprintf("%s:%d", url.QueryEscape(cfg.Host), cfg.Port)
```

Comment:
Tests are failing:

`panic: dial tcp: lookup 127.0.0.1:5432127.0.0.1:5432: no such host`

Try removing this line

Reply:
Also can you add a test?